nettacker/core/graph.py

```diff
@@ -182,10 +284,22 @@ def create_report(options, scan_id):
         with open(report_path_filename, "w", encoding="utf-8") as report_file:
             report_file.write(html_table_content + "\n")
             report_file.close()
```

Comment:
I know it's not your code but could you clean those closes up for consistency? And use `Path.open` too?
Thank you!